runelite-client/src/main/java/net/runelite/client/plugins/party/data/PartyTilePingData.java

```diff
@@ -38,4 +38,5 @@ public class PartyTilePingData
 	private final WorldPoint point;
 	private final Color color;
 	private int alpha = 255;
+	private final long creationTime = System.currentTimeMillis();
```

Comment:
Let's use `System.nanoTime()` instead of `System.currentTimeMillis()` to avoid the potential footgun of a user's system clock adjusting mid-ping, which could leave a ping visible for much longer than intended.